examples/ads.amp.html

```diff
@@ -401,6 +402,13 @@ <h2 id="atomx">Atomx</h2>
       data-id="1234">
   </amp-ad>
 
+  <h2 id="bidtellect">Bidtellect</h2>
```

Comment:
That's why. Please do not put random numbers. We need this ad example for testing reason. Please provide a test ad example that can be rendered.